src/h5web/vis-packs/core/utils.ts

```diff
@@ -349,7 +356,7 @@ export function getValidDomainForScale(
     return [0, max];
   }
 
-  return [min, max];
+  return Number.isFinite(min) ? [min, max] : undefined;
```

Comment:
The way these `Number.isFinite` checks are used is confusing, because it looks like you're checking if `min` is equal to `-Infinity` or if `max` is equal `+Infinity`, whereas it's actually the opposite. Moreover, `min` and `max` are either both equal to `+/-Infinity` or neither of them is, so the check for `Number.isFinite(min)` is superfluous.

In a way it would be clearer to stick with the idea that `getBounds` should return `undefined` when there are no supported values in the array. You can still remove the `values.length === 0` check, but maybe instead of directly returning the result of the `reduce`, you could instead add a ternary and return `undefined` if the `min` is `Infinity`:

```tsx
const bounds = values.reduce(...);
return Number.isFinite(bounds.min) ? bounds : undefined; // return `undefined if `values` has no finite values
```

Reply:
Good call :nerd_face: